README.md

```diff
@@ -12,11 +12,44 @@ cd $GOPATH/src/sdx-encrypted-submitter
 go get -u github.com/streadway/amqp
 go get -u gopkg.in/yaml.v2
 go build
+go install
```

Comment:
Should be able to just do a `go get .` in the repo to pull the dependencies.
Ideally should be be vendoring (see `dep`)

Reply:
I planned on doing this , but I would like to do it on a new branch after this has been merged so that I have a good starting point